src/main/java/org/xstefank/whitelist/RetestCommand.java

```diff
@@ -0,0 +1,35 @@
+package org.xstefank.whitelist;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.xstefank.model.PersistentList;
+
+public class RetestCommand implements Command {
+
+    public static final String NAME = "RetestCommand";
+
+    private String commandRegex;
+
+    @Override
+    public void process(JsonNode payload, PersistentList adminList,
+                        PersistentList userList) {
+        String pullRequestAuthor = WhitelistProcessing.getPRAuthor(payload);
+        String commentAuthor = WhitelistProcessing.getCommentAuthor(payload);
+
+        if (userList.hasUsername(pullRequestAuthor) &&
+                userList.hasUsername(commentAuthor) ||
+                adminList.hasUsername(commentAuthor) &&
+                userList.hasUsername(pullRequestAuthor)) {
```

Comment:
`userList.hasUsername(pullRequestAuthor)` needs to be true in both cases, I think you can rewrite this if 
 clause in a simpler way